src/screens/MessageScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Image, FastImageSource } from 'components/Image';
+import { ButtonProps } from 'react-native-elements';
+import { Text, View, StyleSheet } from 'react-native';
+import { useNavigationParam } from 'react-navigation-hooks';
+import { Button } from 'components/Button';
+
+import { typography, palette } from 'styles';
+
+export interface MessageProps {
+  title: string;
+  source: FastImageSource;
+  description: string;
+  buttonProps?: ButtonProps;
+}
+
+export const MessageScreen = () => {
```

Comment:
I was thinking that you would create just `MessageScreenTemplate` and then in the actual screen, like `AddCustomerSucessScreen` you would use it.

However your approach is also good 👍 even less code. Slightly less verbose. A bit worse for analytics ( All message screen are with the same name for analytics) but there's no one correct way.